content/banzaicloud/drone-kaniko/index.md

```diff
@@ -0,0 +1,142 @@
+---
+date: 2019-01-08T07:27:08-05:00
+title: Kaniko
+author: banzaicloud
+logo: term.svg
+tags: [ publish, docker ]
+repo: banzaicloud/drone-kaniko
+image: banzaicloud/drone-kaniko
+---
+
+This plugin can be used to build and publish images to the Docker registry,
+using the [Kaniko](https://github.com/GoogleContainerTools/kaniko) image
+builder.  Unlike the [Docker plugin](../../drone-plugins/drone-docker) you
```

Comment:
Please use Hugo references to link to this page

Reply:
fixed!